content/how-to/account-management/account-management-faq/index.md

```diff
@@ -51,54 +50,43 @@ Per-billing and per-user usage are not offered at this time.
 
 {{< accordion title="Which users can create support tickets?" col="in" href="accordion22" >}}
 
-All users can create support tickets. However, only the account owner is
-updated on the ticket status.
+All users can create support tickets. However, only the account owner is updated on the ticket status.
```

Comment:
The above would probably be incorrect. To be more precise the answer could be as follows:
`All users with account administration permissions are allowed to create support tickets related to account or device changes. Users limited to device permissions are only allowed to create tickets for the specific devices they are granted permissions on. For further information about Ticket Creation, please visit the Fabric Ticketing section.`

You can link over to the Fabric Ticketing Create Ticket section if possible.